database/migrations/2023_03_27_161300_update_votes_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('votes', function (Blueprint $table) {
+            $table->boolean('isFake');
```

Comment:
Did you resolved that initial default value? Will this migration work in prod?

Reply:
Empty -> means false